x/bridge/keeper/assets.go

```diff
@@ -0,0 +1,60 @@
+package keeper
+
+import (
+	"fmt"
+
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/v23/x/bridge/types"
+)
+
+type ChangeAssetStatusResult struct {
+	OldStatus types.AssetStatus
+	NewStatus types.AssetStatus
+}
+
+func (k Keeper) ChangeAssetStatus(
+	ctx sdk.Context,
+	asset types.Asset,
+	newStatus types.AssetStatus,
+) (ChangeAssetStatusResult, error) {
+	// get current params
+	params := k.GetParams(ctx)
+
+	// check if the specified asset is known
+	const notFound = -1
+	var found = notFound
+	for i := range params.Assets {
+		if params.Assets[i].Asset == asset {
+			found = i
+		}
+	}
+	if found == notFound {
+		return ChangeAssetStatusResult{}, errorsmod.Wrapf(types.ErrInvalidAsset, "Asset not found")
+	}
+
+	// update found asset status
+	oldStatus := params.Assets[found].AssetStatus
+	params.Assets[found].AssetStatus = newStatus
```

Comment:
Can we change "found" to something like "assetIndex"? Took me a second to grok

Reply:
Done